Adyen/Service/StoredValue.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Adyen.Model.StoredValue;
+
+namespace Adyen.Service
+{
+    public class StoredValue : AbstractService
+    {
+        private readonly string _baseUrl;
```

Comment:
```suggestion
        private static final string baseUrl = client.Config.StoredValueEndpoint + "/" + Constants.ClientConfig.StoredValueVersion;;
```

Reply:
This doesn't work actually, as the client.config is passed from the constructor and thus cannot be initialised beforehand..